src/models/BrowserStyleSheet.js

```diff
@@ -113,7 +113,12 @@ export default {
     const names = {}
 
     /* Construct existing state from DOM */
-    Array.from(document.querySelectorAll(`[${SC_ATTR}]`)).forEach(el => {
+    const nodes = document.querySelectorAll(`[${SC_ATTR}]`)
+    const nodesLength = nodes.length
+
+    for (let i = 0; i < nodesLength; i += 1) {
```

Comment:
There shouldn't be :) it's just a tiny bit faster and less hackier than converting that into an array